plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -1553,6 +1561,37 @@ SELECT
 
 `
 
+const sqlServerDiskSpaceV2 string = `
+/* Only for on-prem version of SQL Server
+Gets data about disk space, only if the disk is used by SQL Server
+EngineEdition:
+1 = Personal or Desktop Engine
+2 = Standard
+3 = Enterprise
+4 = Express
+5 = SQL Database
+6 = SQL Data Warehouse
+8 = Managed Instance
+*/
+IF SERVERPROPERTY('EngineEdition') NOT IN (5)
+	BEGIN
+	SELECT DISTINCT
+		'sqlserver_disk_space' AS [measurement]
+		,SERVERPROPERTY('machinename') AS [server_name]
+		,REPLACE(@@SERVERNAME,'\',':') AS [sql_instance]
+		,IIF( RIGHT(vs.[volume_mount_point],1) = '\'	/*Tag value cannot end with \ */
```

Comment:
RTRIM might be better here

Reply:
that won't solve the problem, in fact, there shouldn't be extra spaces on the right.
I've run some tests and discovered that if a tag value ends with "\\" (backslash) then an error will occur, in fact (for reasons unknown to me) it won't be escaped causing a format error in the line protocol. (I've opened an issue in the influx docs about it).
The logic checks if the string ends with that char and removes it. The value is usually the disk path ie: "C:\"